libs/application/templates/old-age-pension/src/lib/OldAgePensionTemplate.ts

```diff
@@ -121,6 +122,10 @@ const OldAgePensionTemplate: ApplicationTemplate<
             },
           },
           progress: 0.25,
+          onExit: defineTemplateApi({
+            action: 'sendApplication',
```

Comment:
Búa til enum í `constants.ts` sem heldur utan um actions
```typescript
export enum ApiModuleActions {
  sendApplication = 'sendApplication',
}
``` 
```suggestion
            action: ApiModuleActions.sendApplication,
```